Makefile

```diff
@@ -134,9 +134,6 @@ endif
 	
 	kubectl cluster-info
 
-	@echo "deploying controller to cluster"
-	make deploy-controller
```

Comment:
What's the reason for removing `make deploy-controller`?

Reply:
Hey sorry should have explained this one. I removed this one as the startup task for the container is calling `make set-kindcluster` and found a lot of the time I didn't really want it to do a deployment each time I started, a lot of the time I wanted to run under debugging against the cluster. 

Happy to put this back if you'd prefer

Reply:
Yes, please put this back. If users enable webhooks `go run ./main.go` won't work and the only way of testing the controller locally is by deploying to cluster.

Reply:
Done.